genie-agent/src/main/java/com/netflix/genie/agent/cli/JobFolderCleanupOption.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+/**
+ * Clean up options for handling the job folder on job completion.
+ *
+ * @author standon
+ * @since 4.0.0
+ */
+public enum JobFolderCleanupOption {
+    /**
+     * Delete the entire job folder.
```

Comment:
We probably need to make sure it's clear this means only on local disk as some contents of the job directory should have already been moved from the agent to the service for backup.